main/src/cgeo/geocaching/connector/gc/GCWebAPI.java

```diff
@@ -811,6 +812,9 @@ static SearchResult searchCaches(final WebApiSearch search) {
         }
 
         result.addAndPutInCache(foundCaches);
+        if (includeGcVote) {
+            GCVote.loadRatings(foundCaches);
```

Comment:
`GCVote.loadRatings` checks for the Settings (whether to load GCVote or not).before issuing any requests to the server.